storops/unity/resource/port.py

```diff
@@ -171,6 +185,37 @@ def get_resource_class(cls):
         return UnityEthernetPort
 
 
+class UnityLinkAggregation(UnityResource):
+    @classmethod
+    def create(cls, cli, ports, mtu=None):
+        resp = cli.post(cls().resource_class,
+                        ports=ports,
+                        mtuSize=mtu)
+        resp.raise_if_err()
+        return cls(_id=resp.resource_id, cli=cli)
+
+    def modify(self, mtu=None, add_ports=None, remove_ports=None):
+        if mtu is None and not add_ports and not remove_ports:

```

Comment:
The client should ensure at least one thing to modify. It may be not necessary to check here.